src/app/downloads/downloads-table.component.css

```diff
@@ -0,0 +1,49 @@
+:host {
+  width: 100%;
```

Comment:
do both :host and table need width and overflow-x? Is it possible for the table to be relative instead? (I can see wanting to put the Export to CSV link in here above the table.)